libbeat/ml-importer/importer.go

```diff
@@ -4,14 +4,32 @@ package mlimporter
 import (
 	"encoding/json"
 	"fmt"
+	"io"
 	"io/ioutil"
+	"net/url"
+	"strings"
 
+	"github.com/joeshaw/multierror"
 	"github.com/pkg/errors"
 
 	"github.com/elastic/beats/libbeat/common"
 	"github.com/elastic/beats/libbeat/logp"
 )
 
+var (
+	esDataFeedURL        = "/_xpack/ml/datafeeds/datafeed-%s"
+	esJobURL             = "/_xpack/ml/anomaly_detectors/%s"
+	kibanaGetModuleURL   = "/api/ml/modules/get_module/%s"
+	kibanaRecognizeURL   = "/api/ml/modules/recognize/%s"
+	kibanaSetupModuleURL = "/api/ml/modules/setup/%s"
+
+	// AvailableModules is the list of modules available in ML
+	AvailableModules = map[string]string{
```

Comment:
What's the purpose for the hard-coded module names inside of libbeat code? I wouldn't expect this data to be part of the library.

Reply:
Yes, I left it accidentaly here. Moving to Filebeat.